app/models/appointment_summary.rb

```diff
@@ -0,0 +1,31 @@
+class AppointmentSummary
+  include ActiveModel::Model
+
+  attr_accessor :supplementary_benefits, :supplementary_debt, :supplementary_ill_health,
+                :supplementary_defined_benefit_pensions,
+                :appointment_type, :format_preference
```

Comment:
Does `format_preference` need to be part of the public interface given we're not using it?